packages/components/src/components/hds/form/field-group/index.hbs

```diff
@@ -0,0 +1,8 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+<Hds::Layout::Flex class={{this.classNames}} @tag="div" @gap="16" @align="end" @wrap={{@wrap}} ...attributes>{{yield
```

Comment:
[TODO] I have think more about this flex and if there's a different way to achieve the same result